.github/CONTRIBUTING.md

```diff
@@ -55,23 +55,23 @@ These tests are run both with the latest available version of `Nextflow` and als
 
 :warning: Only in the unlikely and regretful event of a release happening with a bug.
 
-- On your own fork, make a new branch `patch` based on `upstream/master`.
+- On your own fork, make a new branch `patch` based on `upstream/main` or `upstream/master`.
 - Fix the bug, and bump version (X.Y.Z+1).
-- A PR should be made on `master` from patch to directly this particular bug.
+- Open a pull-request from `patch` to `main`/`master` with the changes.
```

Comment:
Though these changes likely come in from template updates, since references uses the `main` branch, is it worth only mentioning `main` here and above?  Or might it be better to specify `dev`?

Reply:
I missed that one, thanks

Reply:
so the text is coming from the TEMPLATE, I'm not touching the phrasing, but since this repo started with `main` already, I removed all reference to master

Reply:
and this is for patch release, so it should mention main, not dev